errbot/repo_manager.py

```diff
@@ -164,6 +168,14 @@ def get_all_repos_paths(self):
         return [self.plugin_dir + os.sep + d for d in self.get(INSTALLED_REPOS, {}).keys()]
 
     def install_repo(self, repo):
+        """
+        Install the repository from repo
+
+        :repo the url, git url or path on disk of a repository. It can point to either a git repo or
+        a .tar.gz of a plugin
+        :returns the path on disk where the repo has been installed on.
+        :raises RepoException if an error occured.
```

Comment:
Sphinx won't render this correctly, use:

```
:param repo: 
    The url, git url or path on disk of a repository. It can point to either a git repo or
     a .tar.gz of a plugin
:returns:
    The path on disk where the repo has been installed on.
:raises: :class:`~RepoException` if an error occured.
```